mathml/presentation-markup/spaces/space-like-004.html

```diff
@@ -0,0 +1,190 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+<title>Space-like elements</title>
+<link rel="stylesheet" type="text/css" href="/fonts/ahem.css" />
+<link rel="help" href="https://mathml-refresh.github.io/mathml-core/#definition-of-space-like-elements">
+<link rel="help" href="https://mathml-refresh.github.io/mathml-core/#operator-fence-separator-or-accent-mo">
+<link rel="help" href="https://mathml-refresh.github.io/mathml-core/#layout-of-mrow">
+<meta name="assert" content="Verify definition of space-like elements">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<style>
+  /* Default spacing of operator 'X' is 0.2777777777777778em so quite different
+     from the measured/specified 0em and 1em. */
+  math, math * {
+      font: 25px/1 Ahem;
+  }
+  mn {
+      color: black;
+  }
+  mo {
+      color: yellow;
+  }
+  .testedElement, .testedElement * {
+      color: blue !important;
+      background: blue !important;
+  }
+</style>
+<script>
+  function spaceBefore(id) {
+      var element = document.getElementById(id);
+      var mnBeforeParent = element.parentNode.previousElementSibling;
+      return element.getBoundingClientRect().left - mnBeforeParent.getBoundingClientRect().right;
+  }
+
+  function spaceAfter(id) {
+      var element = document.getElementById(id);
+      var mnAfterParent = element.parentNode.nextElementSibling;
+      return mnAfterParent.getBoundingClientRect().left - element.getBoundingClientRect().right;
+  }
```

Comment:
The methods seem to repeat themselves. Can they be moved to some helper script?

Reply:
I was thinking the same, but that seems too specific to these tests to be useful for other situations. Should I do space-like-helper.js for example?